src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/MessageContractTestsVNext.cs

```diff
@@ -0,0 +1,85 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ServiceModel;
+using Infrastructure.Common;
+using WcfService;
+using Xunit;
+
+public static class MessageContractTestsVNext
```

Comment:
Agreed -- choosing the intended version makes things a bit clearer.  At worst, it is no harder to refactor/rename '4.4.0' than it is 'VNext'.  And if the version we choose is correct, there is no cleanup to do when we release.  I've done the refactor/rename in the most recent PR commit